### PR TITLE
Docs: add Gaël Poupard to the core team page

### PR DESCRIPTION
**Pull request by patrickhlauke**

### CODE REVIEW

File: site/data/core-team.yml [hunk 1]
```diff
@@ -27,3 +27,6 @@
 
 - name: Shohei Yoshida
   user: ysds
+
+- name: Gaël Poupard
+  user: ffoodd
```
Comment (XhmikosR) [inline site/data/core-team.yml:32]: Missing EOF line

**Review by XhmikosR — DISMISSED** [review]

File: site/data/core-team.yml (hunk 1 above)
Comment (patrickhlauke) [reply site/data/core-team.yml]: doesn't look like the original had one, unless i'm missing something?

Comment (patrickhlauke) [reply site/data/core-team.yml]: added one now though

Comment (XhmikosR) [reply site/data/core-team.yml]: It did have one :)

PS. please jump on Slack

**Review by mdo — APPROVED** [review]

File: site/data/core-team.yml (hunk 1 above)
Comment (patrickhlauke) [reply site/data/core-team.yml]: yup, seems i missed it somehow, disregard :)

**Review by Johann-S — APPROVED** [review]

**Pull request merged**